### PR TITLE
Fixed project overview not automatically sorting on likes high-low

### PR DESCRIPTION
**Pull request by 1SvenHansen**
## Description
The default sorting is now on Liked high-low projects. Before it was on Created new-old.

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] I updated the changelog with an end-user readable description
-   [x] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

1: Go to the project overview
2: Look at the way the projects are sorted

## Link to issue

Closes: #557

### CODE REVIEW

**Review by waltersajtos — CHANGES_REQUESTED** [review]
Looks good, I don't know if you discussed this with Mees or Dave but personally I would add an entry to the changelog

**Comment by 1SvenHansen** [comment]
@waltersajtos I will ask one of them a quick how-to tomorrow, thanks

**Review by NealGeilen — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,7 +16,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ### Removed
 
 ### Fixed
-
+- Projects overview not sorting on Likes High > Low [#557](https://github.com/DigitalExcellence/dex-frontend/issues/557)
```
Comment (waltersajtos) [inline CHANGELOG.md:19]: I don't like nitpicking, but I think it should be under changes and not fixes since the page was working fine :)

**Review by waltersajtos — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (1SvenHansen) [reply CHANGELOG.md:19]: Fair enough haha

**Review by waltersajtos — APPROVED** [review]

**Review by NealGeilen — APPROVED** [review]

**Pull request merged**